src/generate-changelog.ts

```diff
@@ -1,21 +1,21 @@
 import * as core from '@actions/core'
-import {graphql} from '@octokit/graphql'
+import { graphql } from '@octokit/graphql'
 import shell from 'shelljs'
 
 // Regexes to find the changelog contents within a PR.
-const FIXES_REGEX = /^CHANGELOG-FIXES:(.*)/gm
-const NEW_REGEX = /^CHANGELOG-NEW:(.*)/gm
-const TEAMS_SPECIFIC_REGEX = /^TEAMS-SPECIFIC-CHANGES:(.*)/gm
+const NEW_FEATURE_REGEX = /^CHANGELOG-NEW-FEATURE:(.*)/gm
+const IMPROVEMENT_REGEX = /^CHANGELOG-IMPROVEMENT:(.*)/gm
```

Comment:
Thinking through this more, to ensure we a backwards compatible for a release we need to keep matching the old patterns as well. We should bucket "CHANGELOG-FIXES" into the "bugFixes" bucket and "CHANGELOG-NEW" into the "newFeatures" bucket

Reply:
Ah, that makes sense. I originally thought we could just merge the two PRs near the same time, but I forgot about existing PRs that will still be using the old prefixes. I'll capture both versions here and set a reminder to myself to clean the others up after a few weeks or so.